test/test_distribution.py

```diff
@@ -633,19 +654,20 @@ def gamma(concentration, rate, value):
 
 @pytest.mark.parametrize('batch_shape', [(), (5,), (2, 3)], ids=str)
 @pytest.mark.parametrize('syntax', ['eager', 'lazy'])
+@pytest.mark.xfail(raises=AttributeError, reason="VonMises is not implemented yet in NumPyro")
```

Comment:
nit: unless you're planning to implement `VonMises` in NumPyro, can you change this `xfail` to check if `get_backend() != 'torch'`?